src/schema/schema_worker.c

```diff
@@ -66,6 +66,34 @@ __schema_tiered_worker(WT_SESSION_IMPL *session, const char *uri,
     return (ret);
 }
 
+/*
+ * __schema_layered_worker_verify --
+ *     Run a schema worker operation (which is verification) on the stable table; ingest is not
+ *     supported currently.
+ */
+static int
+__schema_layered_worker_verify(WT_SESSION_IMPL *session, const char *uri,
+  int (*file_func)(WT_SESSION_IMPL *, const char *[]),
+  int (*name_func)(WT_SESSION_IMPL *, const char *, bool *), const char *cfg[], uint32_t open_flags)
+{
+    WT_DECL_RET;
+
+    WT_RET(__wt_session_get_dhandle(session, uri, NULL, NULL, open_flags));
+    WT_LAYERED_TABLE *layered = (WT_LAYERED_TABLE *)session->dhandle;
+    const char *stable_uri = layered->stable_uri;
+
+    WT_ASSERT(session, stable_uri != NULL);
+    WT_ASSERT(session, file_func == __wt_verify);
+
+    WT_WITHOUT_DHANDLE(session,
+      ret = __wt_schema_worker(session, stable_uri, file_func, name_func, cfg, open_flags));
+
+    /* FIXME-WT-15047: Implement ingest table verification */
```

Comment:
Let's make sure people will update the header:
```suggestion
    /* FIXME-WT-15047: Implement ingest table verification and update the header of this function. */
```

Reply:
Done.